src/data/tricks/tricks.schema.json

```diff
@@ -0,0 +1 @@
+{"$ref":"#/definitions/tricks","definitions":{"tricks":{"allOf":[{"type":"object","properties":{"id":{"type":"integer"},"technicalName":{"type":"string","minLength":1},"alias":{"type":"string","minLength":1},"establishedBy":{"type":"string","minLength":1},"yearEstablished":{"type":"integer","exclusiveMinimum":0},"description":{"type":"string","minLength":1},"tips":{"type":"array","items":{"type":"string","minLength":1}},"dateAddedEpoch":{"type":"integer","minimum":0},"videos":{"type":"array","items":{"type":"object","properties":{"link":{"type":"string","minLength":1,"format":"uri"},"startTime":{"type":"number","minimum":0},"endTime":{"type":"number","minimum":0}},"required":["link"],"additionalProperties":false}},"startPosition":{"type":"string","enum":["Stand","Exposure","Korean","Sofa","Chest","Back","Belly","Shoulder","Hang","Nevermind","Dropknee","Double Drop Knee","Inward Drop Knee","Leash","Kneehang","Soup","Sit","Rocket","Frodo","Buddha","Yusus","Banana"]},"endPosition":{"$ref":"#/definitions/tricks/allOf/0/properties/startPosition"},"difficultyLevel":{"type":"integer","minimum":0},"showInSearchQueries":{"type":"boolean"}},"required":["id","technicalName","dateAddedEpoch","startPosition","endPosition","difficultyLevel","showInSearchQueries"]},{"type":"object","properties":{"recommendedPrerequisites":{"type":"array","items":{"type":"integer","minimum":0}},"variationOf":{"$ref":"#/definitions/tricks/allOf/1/properties/recommendedPrerequisites"}}}]}},"$schema":"http://json-schema.org/draft-07/schema#"}
```

Comment:
Yisus instead of Yusus

Reply:
Gets automatically changed on Build if the linked Zod Type gets updated.